README.md

```diff
@@ -33,7 +35,142 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 From within the root directory:
 
 ```sh
-npm install -g webpack
 npm install
+npm install start
+```
+
+To seed database, from within the root directory:
+
+```sh
+npm db:setup
+```
+
+## Server API
+
+### Get place info
+  * GET `/api/places/:placeId`
+
+**Path Parameters:**
+  * `placeId` place id
+
+**Success Status Code:** `200`
+
+**Returns:** JSON
+
+```json
+    {
+        "placeId": "Number",
+        "nightlyFee": "Number",
+        "cleaningFee": "Number",
+        "occupancyTaxRate": "Number",
+        "averateRating": "Number",
```

Comment:
just a spelling thing but I believed you meant "averageRating"?

Reply:
nice catch!! thank you. yes.